README.md

```diff
@@ -1,31 +1,21 @@
 # Smoothed Particle Hydrodynamics (SPH) via Inductiva API
 
-[**Inductiva API**](https://github.com/inductiva/inductiva/tree/main) empowers users to run large-scale simulations. With a simple interface, users have access to several simulators and features that help manage complex simulation workflows. 
+<div align="center">
+<img src="/assets/duck.gif" width=600 height=400 alt="Duck simulation">
+</div>
 
-Here, we explore Smoothed Particle Hydrodynamics (SPH) with two simulators available via **Inductiva API** - [SplishSplash](https://github.com/inductiva/inductiva/wiki/SPlisHSPlasH) and [DualSPHysics](https://github.com/inductiva/inductiva/wiki/DualSPHysics). 
 
-In this repository, we make a tour of **Inductiva API**:
-- [Create a scenario](scenario/README.md);
-- [Generate a dataset via scenario](dataset/README.md).
+With [**Inductiva API**](https://github.com/inductiva/inductiva/tree/main) exploring Smoothed Particle Hydrodynamics (SPH) simulations was never easier. Users can launch the simulations they are familiar with through a simple Python interface. 
 
-### Install
+Two open-source SPH simulators are available via **Inductiva API** - [SplishSplash](https://github.com/inductiva/inductiva/wiki/SPlisHSPlasH) and [DualSPHysics](https://github.com/inductiva/inductiva/wiki/DualSPHysics). On top of them, users can create their custom scenarios and explore a design space with simplicity.
 
-To start experimenting with these scenarios you need to clone the public GitHub repository
+First, don't forget to `pip install inductiva` and then you can follow the [install instructions](INSTALL.md) for this repository to get started.
 
-```bash
-git clone https://github.com/inductiva/inductiva-sph-demos.git
-```
+The **Inductiva API** guide for SPH simulations is the following:
+- [Running a simulation](SPH_SIMULATIONS.md#running-a-simulation);
+- [Create a custom simulation scenario](FLUID_BLOCK.md);
+- [Generate a synthetic dataset at scale](DATASET.md).
```

Comment:
Installation should be the first bullet point. All installation related stuff should go in that section

```suggestion
In this tour, you will learn how to integrate **Inductiva API**  SPH simulation workflows through several examples:
- [How to use this repository and install any dependencies](INSTALL.md);
- [Run a simulation using straightfoward API calls](SPH_SIMULATIONS.md);
- [Leverage the API to create a custom simulation scenario](FLUID_BLOCK.md);
- [Generate a synthetic dataset at scale](DATASET.md).
```